doc/JSON_FLAGS.md

```diff
@@ -1026,34 +1026,40 @@ Some special attacks are also valid use actions for tools and weapons.
 See `monsters.json` for examples on how to use these attacks.
 Also see `monster_attacks.json` for more special attacks, for example, impale and scratch.
 
+- ```ABSORB_MEAT``` Absorbs adjacent meat items (with a monster volume dependent max volume), regenerating health in the process.
```

Comment:
```suggestion
- ```ABSORB_MEAT``` Absorbs adjacent meat items (with a monster volume dependent on max volume), regenerating health in the process.
```

Reply:
What exactly is meant by "with a monster volume dependent on max volume"?

Reply:
The max volume of the item absorbed is calculated depending on the volume of the monster. I'll have to poke this one in particular to see if it could be repurposed into something more general for carnivores down the line.